### PR TITLE
Add missing special skins

### PR DESCRIPTION
**Pull request by Allerek**
Add missing special skins:
- 3, 4, 5, 6, 8, 42, 65, 86, 119, 273, 289

I think i did everything, skins works just fine(not a cutscene model, present in gta3.img)
Tested few animations.
image

### CODE REVIEW

**Comment by Allerek** [comment]
image
Bone count seems fine to me, so its not a cutscene model

**Comment by Allerek** [comment]
Some of the skins have bugged textures, or they're just ugly - but they should be available within MTA.

**Comment by Allerek** [comment]
image
image
image
image
image
image
image
image
image

**Comment by TheNormalnij** [comment]
image
Thanks you! What about [remaining](https://tcrf.net/Grand_Theft_Auto:_San_Andreas/Unused_Characters) ped models?

For next your PR create new branch from master.

**Comment by Allerek** [comment]
> image Thanks you! What about [remaining](https://tcrf.net/Grand_Theft_Auto:_San_Andreas/Unused_Characters) ped models?
> 
> For next your PR create new branch from master.

Rest of them have incorrect bone counts which may break animations or cause crashes
image

File: Client/mods/deathmatch/logic/CClientModelCacheManager.cpp [hunk 1]
```diff
@@ -246,7 +246,7 @@ void CClientModelCacheManagerImpl::ProcessPlayerList(std::map<ushort, float>& ou
         CClientPlayer* pPlayer = *iter;
         ushort         usModelId = (ushort)pPlayer->GetModel();
 
-        if (usModelId < 7 || usModelId > 312)
+        if (usModelId < 1 || usModelId > 313)
```
Comment (patrikjuvonen) [inline Client/mods/deathmatch/logic/CClientModelCacheManager.cpp]: Why was the max changed from 312 to 313?

File: Client/mods/deathmatch/logic/CClientModelCacheManager.cpp [hunk 2]
```diff
@@ -323,7 +323,7 @@ void CClientModelCacheManagerImpl::ProcessPedList(std::map<ushort, float>& outNe
         CClientPed*  pPed = *iter;
         const ushort usModelId = (ushort)pPed->GetModel();
 
-        if (usModelId < 7 || usModelId > 312)
+        if (usModelId < 1 || usModelId > 313)
```
Comment (patrikjuvonen) [inline Client/mods/deathmatch/logic/CClientModelCacheManager.cpp]: Why was the max changed from 312 to 313?

File: Server/mods/deathmatch/logic/CPlayerManager.cpp [hunk 3]
```diff
@@ -336,16 +336,7 @@ bool CPlayerManager::IsValidPlayerModel(unsigned short model)
...
-        case 8:
-        case 42:
-        case 65:
-        case 74:
-        case 86:
-        case 119:
+        case 74: //Missing skin
         case 149:
         case 208:
         case 273:
```
Comment (patrikjuvonen) [inline Server/mods/deathmatch/logic/CPlayerManager.cpp]: 273 and 289 should also be removed.

**Review by patrikjuvonen — CHANGES_REQUESTED** [review]

**Comment by Allerek** [comment]
[skins-test.zip](https://github.com/multitheftauto/mtasa-blue/files/10331355/skins-test.zip)
test resource.

!Important note! - Map editor resource needs new ped ID's added(F1 most likely too)

**Review by patrikjuvonen — APPROVED** [review]
Thanks!

**Comment by Fernando-A-Rocha** [comment]
I think the smoke with bloody vest skin was forgotten.

Comparison: cssmokevest VS cssmokevest2 (found in custscene.img)

img

**Comment by Allerek** [comment]
> I think the smoke with bloody vest skin was forgotten.
> 
> Comparison: cssmokevest VS cssmokevest2 (found in custscene.img)
> 
> img

We do not support cutscene skins.

**Pull request merged**